src/connectionbenchmark/connectionbenchmarkitem.h

```diff
@@ -0,0 +1,39 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef CONNECTIONBENCHMARKITEM_H
+#define CONNECTIONBENCHMARKITEM_H
+
+#include "qmlengineholder.h"
+
+#include <QObject>
+
+class ConnectionBenchmarkItem : public QObject {
```

Comment:
I think we don't need this class. What about if we have a model able to expose just properties such as:
- ping
- download
- update (in future)

In this way, we do not need to have a list, but a simple QObject.